packages/vector_graphics_compiler/lib/src/svg/node.dart

```diff
@@ -125,9 +125,24 @@ class ParentNode extends Node {
   }
 
   @override
-  void addPaths(DrawCommandConcatentor addDrawPath, AffineMatrix transform) {
+  void addPaths(DrawCommandBuilder builder, AffineMatrix transform) {
+    final Color? color = paint?.fill?.color;
+
+    if (color != null && color.a < 255) {
```

Comment:
Should I unconditionally create this layer if there is a fill paint on this group? I'm not really sure about the breakdown of fill/stroke properties.

Reply:
What really matters is the shader, the blend mode, and the color if there is no shader.

That said, I don't think flutter_svg currently ever uses the shader for this purpose.

Maybe we could give the `Paint` object some more to-level properties for save layer purposes, or just decide to always use fill. I think always using fill is fine.

Reply:
But if you have a color, then your children still inherit the non-opaque parts of it, no?

Reply:
You can have a color blended in, whether transparent or opaque